src/components/FormPoll.vue

```diff
@@ -243,10 +254,12 @@ export default {
             "rgba(0, 0, 0, 0.2)";
```

Comment:
Let's solve this using [more vue-ish style](https://vuejs.org/v2/guide/class-and-style.html)